ui/app/components/service-status-bar.js

```diff
@@ -0,0 +1,45 @@
+import { computed } from '@ember/object';
+import DistributionBar from './distribution-bar';
+import { attributeBindings } from '@ember-decorators/component';
+import classic from 'ember-classic-decorator';
+
+@classic
+@attributeBindings('data-test-service-status-bar')
```

Comment:
What are the main reasons to keep this a classic component? Whether or not we do so, would the ergonomics be better for us to keep test attribute bindings in template logic, maybe using `tagname = ""` and treating the template like a more modern component?

Reply:
There's no benefit beyond speed. This component is derived from `DistrbutionStatusBar` and specification on the design requested to have the exact pattern as the `JobStatusBar`. I figured it would be unforgiveable if I made a brand new component instead of using what's already available.